wolfcrypt/src/asn.c

```diff
@@ -3283,6 +3285,8 @@ int EncryptContent(byte* input, word32 inputSz, byte* out, word32* outSz,
     (void)rng;
 
     return totalSz;
+
+    (void)rng;
```

Comment:
This same (void)rng; cast is also above.  This one is not needed, please remove.

Reply:
I removed (void) rng.